applications/ConvectionDiffusionApplication/python_scripts/convection_diffusion_explicit_solver.py

```diff
@@ -0,0 +1,115 @@
+# Importing the Kratos Library
+import KratosMultiphysics
+
+# Import applications
+import KratosMultiphysics.ConvectionDiffusionApplication as ConvectionDiffusionApplication
+
+# Import base class file
+from KratosMultiphysics.ConvectionDiffusionApplication import (
+    convection_diffusion_base_solver,
+)
+
+
+def CreateSolver(model, custom_settings):
+    return ConvectionDiffusionExplicitSolver(model, custom_settings)
+
+
+class ConvectionDiffusionExplicitSolver(
+    convection_diffusion_base_solver.ConvectionDiffusionBaseSolver
+):
```

Comment:
```suggestion
class ConvectionDiffusionExplicitSolver(convection_diffusion_base_solver.ConvectionDiffusionBaseSolver):
```

Reply:
I am using a formatter which automatically splits lines if too long. Is there a default Python formatter? I have not found any in the Style guide.